uxarray/plot/utils.py

```diff
@@ -33,3 +35,30 @@ def reset_mpl_backend(self):
 
 # global reference to holoviews backend utility class
 backend = HoloviewsBackend()
+
+
+def great_circle(p1, p2, num_points=100):
+    """
+    Generate points along the great circle between two points.
+    p1 and p2 are arrays/lists of Cartesian coordinates.
+    """
+    # Normalize the input points
+    p1 = p1 / np.linalg.norm(p1)
+    p2 = p2 / np.linalg.norm(p2)
+
+    # Compute the angle between the points
+    omega = np.arccos(np.clip(np.dot(p1, p2), -1.0, 1.0))
+
+    if omega == 0:
```

Comment:
probably better to use `is_closed` with the error tolerance than using the equal sign